build/steps-build.yml

```diff
@@ -112,6 +112,15 @@ steps:
     codeCoverageTool: 'Cobertura'
     summaryFileLocation: "$(Build.SourcesDirectory)/src/coverage.cobertura.xml"
 
+- task: nventiveReleaseNotesCompiler@5
```

Comment:
Is this task public? Is it going to work for people outside the nventive organization?

Reply:
No, it's not. It could easily be made pubic though, given its simplicity.